scripts/ci/deploy.sh

```diff
@@ -19,18 +19,37 @@
 set -e
```

Comment:
Perhaps this script should either be renamed to `deploy-aws-s3.sh`, or it should also make the `publish-bintray-debian` target?